src/main/java/com/neuvector/Scanner.java

```diff
@@ -329,6 +362,74 @@ private static String maskCredential(String message, String credential){
         return message.replace(credential, "******");
     }
 
+    static String getDockerUserGroupCmdArg(String scanReportPath) {
+        // No user arg if file exists, and it is owned by root
+        Boolean ownedByRoot = ownedByRoot(scanReportPath);
+        if (ownedByRoot != null && ownedByRoot) {
+            return null;
+        }
+
+        String cmdUserGroupArg = null;
+        String UID = executeCommand("id -u");
+        if ((UID != null && !UID.isEmpty())) {
+            if ("0".equals(UID)) {
+                return null; // runs as root
+            }
+            String GID = executeCommand("stat -c '%g' /var/run/docker.sock");
```

Comment:
This is an important change: the group ID used is the `/var/run/docker.sock` group ID